webpack/assets/javascripts/react_app/redux/middlewares/ajaxRequestAction.js

```diff
@@ -0,0 +1,31 @@
+import API from '../../API';
+import { API_OPERATIONS } from '../consts';
+import actionTypeGenerator from '../actionTypeGenerator';
+
+const get = (payload, url, store, actionTypes) => {
+  store.dispatch({
+    type: actionTypes.REQUEST,
+    payload,
+  });
+  API.get(url, payload.headers || {}, payload.params || {})
+    .then(({ data }) => {
```

Comment:
can you please use `async` - `await` syntax ?

Reply:
Changed, thanks